website/pages/partials/builders/aws-session-manager.mdx

```diff
@@ -2,11 +2,13 @@
 
 Support for the AWS Systems Manager session manager lets users manage EC2 instances without the need to open inbound ports, or maintain bastion hosts. Session manager connectivity relies on the use of the [session manager plugin](#session-manager-plugin) to open a secure tunnel between the local machine and the remote instance. Once the tunnel has been created all SSH communication will be tunneled through SSM to the remote instance.
 
+Please note that session manager connectivity is currently only implemented for the SSH communicator, not the WinRM Communicator.
+
```

Comment:
Suggestion: This could probably be made into a note so people can see it better:
```suggestion
->Note: Please note that session manager connectivity is currently only implemented for the SSH communicator, not the WinRM Communicator.

```

Reply:
Good call! Updating now.